app/ui-react/syndesis/src/modules/data/locales/data-translations.en.json

```diff
@@ -15,7 +15,6 @@
     "RUNNING": "Running",
     "SUBMITTED": "Submitted"
   },
-  "connectionNotFound": "Connection not found",
```

Comment:
Just to double-check do you intend for these lines to be removed?

Reply:
reverted all but cursor related entries